### DISCUSSION TITLE
Better zooming

### DISCUSSION THREAD

**Discussion by RoniJacobson** [d2phap/ImageGlass#823] [General] [answered]
### Is your feature request related to a problem? Please describe.

When comparing ImageGlass to other image viewers I noticed that zooming is of significantly lower quality in ImageGlass, and can become quite blurry quite quickly (esp. when dealing with small text)
### Describe the solution you'd like

I would like the zooming in algorithm to be improved
### Additional context

Windows photos app:
![image](https://user-images.githubusercontent.com/36922833/87587522-e089de00-c6ea-11ea-8a12-7f16a0276722.png)
ImageGlass
![image](https://user-images.githubusercontent.com/36922833/87587557-e8e21900-c6ea-11ea-8e5a-c7c98ffdd7f5.png)

**Comment by d2phap** [comment, marked as answer]
Hi @RoniJacobson 
You can change this in Settings > Image > Zoom optimization to Smooth pixels
